torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3093,7 +3071,7 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            skips=(
                # "cumprod_cuda" not implemented for 'BFloat16'
                SkipInfo('TestOpInfo', 'test_supported_backward', dtypes=(torch.bfloat16,)),
-               # prod does not correctly warn when resizing out= inputs
+               # prod does not support the (Tensor, *, out) overload
```

Comment:
That is weird: https://pytorch.org/docs/master/generated/torch.prod.html?highlight=prod#torch.prod.

That's actually a UX bug.